hack/get_ci_secrets.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# This script fetches the secrets from the CI cluster and populates local files with it.
+# This script is meant to be sourced.
```

Comment:
Can we mention that this file is mostly for the local development to set up the environment that exists in CI?

Reply:
In its current form it would be hard to comprehend with respect to its use by anyone who is new to Cincinnati.

Reply:
To me it's hard to comprehend how we would rely on anything in a directory called `hack` for anything other than local development environments :wink: I did change the description though. PTAL